DigitalLearningSolutions.Data/Services/CourseContentService.cs

```diff
@@ -56,7 +56,9 @@ FROM Applications
                   INNER JOIN Customisations ON Applications.ApplicationID = Customisations.ApplicationID
                   INNER JOIN Sections ON Sections.ApplicationID = Applications.ApplicationID
                   INNER JOIN Centres ON Customisations.CentreID = Centres.CentreID
-                  LEFT JOIN Tutorials ON Sections.SectionID = Tutorials.SectionID
+                  INNER JOIN Tutorials ON Sections.SectionID = Tutorials.SectionID
+                  INNER JOIN CustomisationTutorials ON Customisations.CustomisationID = CustomisationTutorials.CustomisationID
+                                                   AND Tutorials.TutorialID = CustomisationTutorials.TutorialID

```

Comment:
We also need to ensure that 
```
Sections.ArchivedDate != NULL && 
(CustomisationTutorials.Status == 1 || CustomisationTutorial.DiagStatus == 1 || Customisations.IsAssessed)
```
(just pseudocode to show the logic)

Reply:
I've added those in fd5e084 🙂

I've been using `Sections.ArchivedDate IS NULL` instead of `IS NOT NULL`, which I think was what you meant 🙂